src/website/src/pages/ec/help/faq/index.md

```diff
@@ -0,0 +1,143 @@
+---
+order: 1
+title: 'FAQ'
+section: 'Help'
+---
+
+## What is ECL?
+
+ECL stands for "Europa Component Library".
+
+The purpose of the ECL is to ensure, in a cost-effective manner, one coherent ‘Europa’ environment and reinforce visitors' confidence in the information provided.
+
+The ECL ensures this coherence by providing a comprehensive, modular design system describing and showcasing the design elements and visual standards that make up all EU and EC branded websites and web information systems.
```

Comment:
Vast question: should we say "ECL" or "the ECL"?

Reply:
Suggestion: i think we should use ECL and if we are referring to the ECL we could use this component library.  For example: 
 - The purpose of ECL is to ensure, in a cost-effective manner, one coherent ‘Europa’ environment and ... 
-  This component library ensures this coherence by providing a comprehensive, modular design system describing and showcasing ....